### PR TITLE
fixed typo in config variable

### PR DESCRIPTION
**Pull request by wodCZ**

### CODE REVIEW

**Comment by northys** [comment]
@wodCZ :+1:

File: README.md [hunk 1]
```diff
@@ -172,4 +172,4 @@ class AppExtension extends CompilerExtension implements IAssetsProvider
 * 1.0.2
 	+ Optimized dependencies
 	+ Removed unused exceptions
-	+ Optimized checking for files' modifications
\ No newline at end of file
+	+ Optimized checking for files' modifications
```
Comment (davidkudera) [inline README.md:175]: Just curious, what change is at that line? :-)

Comment (davidkudera) [inline README.md:175]: And now I just noticed that whole changelog uses wrong order..

Comment (wodCZ) [inline README.md:175]: Not sure what caused that. I've used Github editor (it was just typo, so I didn't want to clone whole project)

**Comment by davidkudera** [comment]
thank you

**Pull request merged**